crates/starknet-devnet/src/cli.rs

```diff
@@ -198,10 +199,13 @@ Sending POST /create_block is also an option in modes other than \"demand\".")]
     #[arg(default_value_t = DEVNET_DEFAULT_REQUEST_BODY_SIZE_LIMIT)]
     request_body_size_limit: usize,
 
-    #[arg(long = "disable-account-impersonation")]
-    #[arg(env = "DISABLE_ACCOUNT_IMPERSONATION")]
-    #[arg(help = "Disables the possibility to impersonate accounts;")]
-    disable_account_impersonation: bool,
+    #[arg(long = "restrictive-mode")]
+    #[arg(env = "RESTRICTIVE_MODE")]
+    #[arg(num_args = 0..)]
+    #[arg(help = "Use devnet in restrictive mode; You can specify the methods that will be \
+                  fobidden (empty space separated values). If nothing is specified for this \
+                  argument, then default restricted methods are used.")]
```

Comment:
- Devnet should be capitalized.
- "fobidden"
- What are "empty space separated values"?
- How will the user know the default restricted methods? Perhaps this could include a link to that paragraph in the docs (we can predict the exact paragraph link, even though it might not yet exist before merging).

Reply:
I thought you meant that, but I think that "empty space separated" is more commonly expressed as "whitespace-separated" (with the hyphen)